document/cloud-client/batch_parse_form_test.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific ladnguage governing permissions and
+# limitations under the License.
+
+import os
+import batch_parse_form
+
+PROJECT_ID = os.environ["GCLOUD_PROJECT"]
+INPUT_URI = "gs://cloud-samples-data/documentai/invoice.pdf"
+OUTPUT_URI = os.environ["BATCH_OUTPUT_URI"]
+
+
+def test_batch_parse_form(capsys):
+    batch_parse_form.batch_parse_form(PROJECT_ID, INPUT_URI, OUTPUT_URI)
+    out, _ = capsys.readouterr()
+    assert "Output files" in out
```

Comment:
Add a teardown to delete the created filed.

Reply:
Example: (not of the storage though)
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/automl/beta/video_classification_create_dataset_test.py

Reply:
Ah here's one: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/vision/cloud-client/detect/vision_async_batch_annotate_images_test.py

Reply:
I updated this to create a bucket for each test run. PTAL